File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int arr[26]={0};
        for(char c:s)
        {
            arr[c-97]++;
        }
        for(char c:t)
        {
            arr[c-97]--;
            if(arr[c-97]==-1)
            {
                return false;
            }
        }
        for(int value:arr)
        {
            if(value!=0)
            {
                return false;
            }
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        arr=[0]*26
        for char in s:
            arr[ord(char)-97]+=1
        for char in t:
            arr[ord(char)-97]-=1
        for i in range(26):
            if arr[i]!=0:
                return False
        return True